internal/api/provider/oidc.go

```diff
@@ -312,6 +314,44 @@ func parseAzureIDToken(token *oidc.IDToken) (*oidc.IDToken, *UserProvidedData, e
 	return token, &data, nil
 }
 
+type KakaoIDTokenClaims struct {
+	jwt.StandardClaims
+
+	Email         string `json:"email"`
+	EmailVerified bool   `json:"email_verified"`
```

Comment:
This is not listed here: https://developers.kakao.com/docs/latest/en/kakaologin/common#id-token

It appears that the email in Kakao is always verified?

Please also extract the `nickname` and all other custom claims too.

Reply:
Yes, I will check and fix it.

Reply:
```
Email saved in Kakao Account.
Only returned is_email_valid is true.
```

https://developers.kakao.com/docs/latest/en/kakaologin/rest-api#get-id-token-info-response-body